src/push-notifications.js

```diff
@@ -2,7 +2,7 @@ import doRequest from './doRequest';
 import TokenProvider from './token-provider';
 import DeviceStateStore from './DeviceStateStore';
 
-const DEFAULT_SERVICE_WORKER_URL = '/service-worker.js';
+const SERVICE_WORKER_URL = '/service-worker.js?pusherBeamsWebSDKVersion=1.6.0';
```

Comment:
Apparently we are in version `1.6.0`. I didn't want to diverge here.

Is there a good way to keep this version and the one in the package.json in sync?

Reply:
You can actually import the package json as a module (via rollup) I believe